hack/lib.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
It's intended to be generic, for it to be included all the things that's reusable. I really don't want to include multiple files in bash with its non-existent dependency management system.

Reply:
Please move these variables into the Makefile and remove this bash include for now.
All our scripts are intended to be only called via make and we should not start putting our variables in multiple places.

If we need shared bash code in multiple scripts, we can revisit such a `lib.sh`, although I would rather have longer scripts with different command args than using includes in bash.

Reply:
It's included in two places, and I'd rather have those things defined in one place:

* test.Dockerfile building
* proto generation

I don't want c/p the constants around without the good need for it.

Reply:
The `testDockerfile` isn't built via calling the make (internally, it just c/p a few bash files and runs them) since it would entail copying bunch of files and recreating the folder structure the make expects...and I'd rather depend on a few files as possible while building the `test.Dockerfile`